Competition2017/.classpath

```diff
@@ -12,5 +12,7 @@
 	<classpathentry kind="lib" path="/SeriouslyCommonLib/lib/WPILib-sources.jar"/>
 	<classpathentry kind="lib" path="/SeriouslyCommonLib/lib/WPILib.jar" sourcepath="/SeriouslyCommonLib/lib/WPILib-sources.jar"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="lib" path="lib/influxdb-java-2.5.jar" sourcepath="/home/alex/frc/repos/influxdb-java"/>
```

Comment:
good call, we shouldn't include the sourcepath in here at all, just needed it for some initial debugging